pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTestSet.java

```diff
@@ -146,6 +146,9 @@ private void testHardcodedQueriesCommon()
     String query;
     String h2Query;
 
+    // SUM result will overflow INTEGER
+    query = "SELECT SUM(ActualElapsedTime) FROM mytable";
```

Comment:
should at least add 4 tests for INT LONG(BIGINT) FLOAT(REAL) and DOUBLE

Reply:
done